plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -16,39 +16,37 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
+                                                                                    // {Relu,        {}},  // TODO
+                                                                                    {Exp, {{}}},
+                                                                                    {Log, {{}}},
+                                                                                    {Sign, {{}}},
+                                                                                    {Abs, {{}}},
+                                                                                    {Clamp, {{-2.0f, 2.0f}}},
+                                                                                    {Negative, {{}}},
+                                                                                    {Acos, {{}}},
+                                                                                    {Acosh, {{}}},
+                                                                                    {Asin, {{}}},
+                                                                                    {Asinh, {{}}},
+                                                                                    {Atan, {{}}},
+                                                                                    {Atanh, {{}}},
+                                                                                    {Cos, {{}}},
+                                                                                    {Cosh, {{}}},
+                                                                                    {Floor, {{}}},
+                                                                                    {Sin, {{}}},
+                                                                                    {Sinh, {{}}},
+                                                                                    {Sqrt, {{}}},
+                                                                                    {Tan, {{}}},
+                                                                                    {Elu, {{0.1f}}},
+                                                                                    {Erf, {{}}},
+                                                                                    {HardSigmoid, {{0.2f, 0.5f}}},
+                                                                                    {Selu, {{1.6732f, 1.0507f}}},
+                                                                                    {Ceiling, {{}}},
+                                                                                    {Swish, {{1.0f}}},
+                                                                                    {Mish, {{}}},
+                                                                                    {HSwish, {{}}},
+                                                                                    {SoftPlus, {{}}}};
```

Comment:
Add a trailing comma here to make it format nicer.

Reply:
Oh cool thanks! I was trying various things and failing